pandas/tests/extension/test_integer.py

```diff
@@ -193,7 +197,20 @@ class TestGetitem(base.BaseGetitemTests):
 
 
 class TestSetitem(base.BaseSetitemTests):
-    pass
+    def test_setitem_series(self, data, full_indexer):
```

Comment:
Can you indicate here why it is overriding the base class?

Reply:
comment added